docs/semgrep-ci/configuration-reference.md

```diff
@@ -19,6 +19,58 @@ While environment variables are the preferred way to configure Semgrep CI, pass
 SEMGREP_RULES="p/security-audit p/secrets"
 ```
 
+## Suppressing blocking comments or errors
+
+Semgrep in CI can block a pull request (PR) or a merge request (MR) from being merged. This block can be caused by Semgrep's blocking comment or internal error of Semgrep. One option to avoid blocked PRs and MRs is to change configuration of blocking comments in the rule board of Semgrep App. Another option that Semgrep in CI allows you is to disable blocking comments without changing the setup of rules in the rule board.
```

Comment:
> can be caused by Semgrep's blocking comment or internal error
It's less a blocking comment and semgrep returning exit code 1 when it has a blocking finding that causes CI providers to potentially block a PR